samples/kind-lb/setupkind.sh

```diff
@@ -8,7 +8,7 @@ set -e
 # This script can only produce desired results on Linux systems.
 ENVOS=$(uname 2>/dev/null || true)
 if [[ "${ENVOS}" != "Linux" ]]; then
-  echo "Your environment is not supported by this script."
+  echo "Your environment is not supported by this script. Environments supported by this script: Linux"
```

Comment:
I think the following maybe a bit better:

```
echo "Your system is not supported by this script. Only Linux is supported"
```

Reply:
Updated.